app/client/src/tests/unit/App.test.js

```diff
@@ -0,0 +1,153 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
The name of the file is App.test.js, but based on your documentation, do we want to refine this? ie// Login.test.js

Reply:
That would be better. Isolate just the login portion. Currently it renders everything in App. I expect these to change as the front-end develops further.